frontend/src/renderer/util/photoViewerHelper.js

```diff
@@ -0,0 +1,11 @@
+
+// Author: Elijah Cooper
+// z = (x - mix(x)) / (max(x) - min(x)) * 100     (how to scale between 0 and 100)
+// the getZoom method returns the ratio of the image's width to the width of the viewport.
+// that number has a logarithmic behavior when zooming in.
+// so we can simply take the log of that number and scale it between 0 and 100.
+export function getZoomPercentage(currentZoom) {
+    let logZoom = (currentZoom / 59) * 100
+    logZoom = Math.log(logZoom)
+    return ((logZoom - 0.52763274) / (4.07168653 - 0.52763274)) * 100
+}
```

Comment:
@egc2846 extracted this to a new file, I'd like to get confirmation from you that this is good. Were we not planning on having the zoom-percentage be more standard like when zooming in with a browser? (it may be simple like (width of entire view)/(width of what you're currently seeing)

Reply:
Width of entire / width of current view is what the getZoom function returns i believe, the only thing we were talking about changing was making it start at 100% and then go up from there instead of zero, to my memory
also thank you youre a legend